Testnet/StratisSwap/StratisSwap/JsonConfig/JsonConfig.cs

```diff
@@ -0,0 +1,157 @@
+﻿using Stratis.SmartContracts;
+
+[Deploy]
+public class JsonConfig : SmartContract
```

Comment:
Where is this being used ?

Reply:
In the client, so different versions can have access to basic dynamic configurations.